feat: Comment Aggregate Mock Api

yeonju73 opened the pull request:
## 🐣Title
feat: Comment 작업중


## 🐣Part
BE


## 🐣Key Changes

dto 패키지 - dto 추가 개발
> CommentUpdate / CommetLikeUpdate

api 패키지 - controller 수정

--------------------------------------------------

application < CommentService 구현

- createComment에서 Post와 User 각각 id로 find하는 부분 연결 필요합니다.

- deleteComment는 임시적으로 부모 댓글이 삭제되면 자식 댓글도 삭제되는 로직으로 구현했습니다. 추후 수정하겠습니다.

dto < CommentUpdateResponse에 text 추가
단일 응답으로 필요한 정보 제공을 위해

domain < Text / LikeCount setter 추가
update 로직 구현을 위해 추가


## 🐣Next

댓글 삭제에 관한 부분 회의 후 수정 또는 확정하겠습니다.
댓글 생성 시 Post와 User 와의 연결 필요
많 은 훈 수 부 탁 드 립 니 다 !! ㅠ_ㅠ
</br>
## 🐣Issue
#12

yeonju73 committed:
feat: Comment 작업중

yunuo46 reviewed (commented):
api에 작성되어있는 request, response dto는 dto패키지로 이동해주세요!

yunuo46 commented on src/main/java/com/gaplog/server/domain/comment/dto/request/CommentRequestDto.java:
이 dto에만 access level을 private으로 설정하신 이유가 있을까요?

yunuo46 commented on src/main/java/com/gaplog/server/domain/comment/domain/Comment.java:
잘하셨어요 !!

yeonju73 committed:
rename: api 하위 dto, dto 패키지로 이동 DTO 패키지 내의 class rename
올바른 패키지로 이동시키고 dto 패키지 내에 있는 dto들 Dto 제거

yeonju73 committed:
feat: response dto 추가, dto 패키지의 request, responce dto 수정
request dto들의 Data 어노테이션 삭제
response dto 세분화

yeonju73 committed:
fix: CommentApi 수정, CommentService 인자 수정
Api 수정하고 Service는 response만 임의로 해주었습니다.

yeonju73 committed:
feat: CommentService 구현
CommentService에 댓글등록, 수정, 삭제, 좋아요 수 수정 기능 구현
서비스 로직 구현을 위해 domain에 setter 추가
deleteComment 임시 구현, 부모댓글 삭제 시 자식 댓글 모두 삭제

yunuo46 committed:
Merge branch 'main' into main

yunuo46 commented on src/main/java/com/gaplog/server/domain/comment/application/CommentService.java:
```diff
@@ -1,38 +1,89 @@
+
+        // like가 true면 like을 새롭게 누른 것, false면 눌렀던 like를 지운 것으로 생각했습니다.
+        if (like){
+            comment.setLike_count(comment.getLike_count() + 1);
```
카멜 케이스와 스네이크 케이스를 혼합해서 사용하지 말아주세요

yeonju73 replied:
확인했습니다 감사합니다!
